.github/workflows/bindings_java.yml

```diff
@@ -84,8 +84,4 @@ jobs:
         # https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-to-test-my-maven-build-reproducibility
         shell: bash
         run: |
-          if [[ "${{ runner.os }}" != Linux ]]; then
-            export NO_DOCKER=true
-          fi
           ./mvnw clean install -DskipTests -Dcargo-build.features=services-redis
-          ./mvnw verify artifact:compare -Dcargo-build.features=services-redis
```

Comment:
We should still verify the reproducible build. But I'll review the main change and follow-up later.